mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx

```diff
@@ -314,109 +329,81 @@ export const Columns = (props: ColumnsProps) => {
+          const updatedComments = row.original.comments.map((item) => {
+            const itemToReturn = item.uuid === comment.uuid
+              ? new Comment({
+                ...comment,
+                description: text,
+              })
+              : item;
 
-            return item;
+            return itemToReturn;
           });
 
-          const updatedDayReport = {...row.original, studentComments: updatedStudentComments};
-          updateDayReportState(props.dayReports, props.setDayReports, updatedDayReport);
-        };
-
-        return (
-          <TableCell
-            buttonValue="add comment"
-            onButtonClick={() => createStudentComment()}
-          >
-            {row.original.studentComments
-              .map((studentComment, index) => (
-                <CellItem key={index}>
-                  <EditableText
-                    text={studentComment}
-                    onChangeFinish={(text) => updateStudentComment(text, index)}
-                  />
-                </CellItem>
-              ),
-              )}
-          </TableCell>
-        );
-      },
-    }),
-    columnHelper.accessor("mentorComments", {
-      header: "Mentor comments",
-
-      /**
-       * Cell with MentorComments items
-       */
-      cell: ({row}) => {
-
-        /**
-         * Create MentorComment
-         */
-        const createMentorComment = async () => {
-          const mentorComment = await MentorCommentDAL.createMentorComment(row.original);
-          const mentorComments = [...row.original.mentorComments, mentorComment];
-          const updatedDayReport = {...row.original, mentorComments};
+          const updatedDayReport = {...row.original, comments: updatedComments};
           updateDayReportState(props.dayReports, props.setDayReports, updatedDayReport);
         };
 
         /**
-         * Update MentorComment
+         * Get user name
          */
-        const updateMentorComment = async (mentorComment: MentorComment, text: string) => {
-          await MentorCommentDAL.updateMentorComment(mentorComment, text);
-          const updatedMentorComments = row.original.mentorComments.map((item) => {
-            if (item.uuid === mentorComment.uuid) {
-              return new MentorComment({
-                ...mentorComment,
-                description: text,
-              });
-            }
-
-            return item;
-          });
-
-          const updatedDayReport = {...row.original, mentorComments: updatedMentorComments};
-          updateDayReportState(props.dayReports, props.setDayReports, updatedDayReport);
+        const getMentorName = (users: Map<string, UserPreview>, uuid: string) => {
+          const mentor = users.get(uuid);
+
+          /**
+           * TODO: need to delete this check after we will add possibility to add comments only for mentors
+           */
+          if (!mentor) {
+            return "User is not mentor";
+          }
```

Comment:
Could it bring new bug to the app?  or it shuold not happend?

Reply:
There are comment about delete it after logic about check you are mentor or not will be added

Reply:
There are comment about delete it after logic about check you are mentor or not will be added